progs/cram_dump.c

```diff
@@ -157,7 +157,8 @@ void dump_name_block(cram_block *b, int verbose) {
 }
 
 void dump_tag_block(cram_block *b, int verbose) {
-    return dump_core_block(b, verbose);
+    dump_core_block(b, verbose);
+    return;
```

Comment:
Doesn't need the return statement at all infact.  I'm not quite sure why I had one there before.  I'll amend during merge.